compiler/optimizer/BenefitInliner.hpp

```diff
@@ -3,7 +3,7 @@
  *
  * This program and the accompanying materials are made available under
  * the terms of the Eclipse Public License 2.0 which accompanies this
- * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/s
```

Comment:
There's an extra `s` at the end of this URL

Reply:
Corrected it.